### PR TITLE
PhysicalMachineChaos: add recover command in process (#3084)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #3084 to release-2.1
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In chaos-mesh repo:
git pr https://github.com/chaos-mesh/chaos-mesh/pull/3468
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/chaos-mesh.git pr/3468:release-2.1-ad48499bbda9
```

---

Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

Since https://github.com/chaos-mesh/chaosd/pull/122 add a `recover-cmd` for process kill, this pr update the API and supported it in Chaos Mesh

### What's changed and how it works?

Add a new field in PhysicalMachineChaos' spec

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [x] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by iguoyr — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 
 ### Added
 
-- Nothing
+- PhysicalMachineChaos: add recover command in process [#3468](https://github.com/chaos-mesh/chaos-mesh/pull/3468)
```
Comment (cwen0) [inline CHANGELOG.md:39]: We should add it on `## [Unreleased]` @WangXiangUSTC

**Pull request merged**